docs/pages/installation/port-forwarding.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Port-Forwarding
+permalink: services/port-forward
+type: services
+display-title: "true"
+language: en
+list: exclude
+---
+<div class="prereqs"><p><strong style="font-size: 20px;">Prerequisites</strong> </p> 
+  <ol>
+    <li>Make sure <a href="https://kubernetes.io/docs/tasks/tools/" class="meshery-light">kubectl</a> is installed on your local machine.</li>
+    <li>You should have access to the cluster/platform where Meshery is deployed.</li>
+    <li>Ensure that the kubeconfig file has the correct current context/cluster configuration.</li>
+  </ol>
+</div>
+
+## Port Forward to Meshery UI
```

Comment:
`mesheryctl system dashboard --port-forward`

Reply:
let me add the reference link of mesheryctl system dashboard page